exts/betterTooltip/betterTooltip.js

```diff
@@ -91,6 +226,17 @@
                 return _seriesLabelFormatFunction;
             };
 
+            /**
+             * Gets or sets the format string for the label of the measures. Used by the default label formater.
```

Comment:
Gets or sets the format string for the label of the measures. Used by the default label formatter.